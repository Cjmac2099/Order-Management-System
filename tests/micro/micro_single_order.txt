WAGZ2AHLHK
13.99
END
