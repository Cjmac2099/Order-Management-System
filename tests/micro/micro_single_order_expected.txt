Loading catalog...
Enter 'END' to finish order.
SKU> 
SKU matches product Juice Apples frozen concentrate, price per pound: $0.61
Quantity>
SKU> 
Printing order summary:
13.99lbs Juice Apples frozen concentrate: $8.51
